tests/test_client.py

```diff
@@ -11,9 +11,9 @@
 from click.testing import CliRunner
 
 try:
-    from unittest.mock import call, patch
+    pass
```

Comment:
This doesn't look right! I think the old construct imported `call` and `patch` mocks from the package that was named differently in py2 vs py3. This does nothing in `try` and then does nothing again if that fails. :-) The package name in py3 is `unittest.mock`, so shouldn't we just keep the one statement, `from unittest.mock import call, patch`?

Reply:
Or, maybe I'm missing something and the import was not needed any more (since CI tests pass?), in which case we can remove this whole try-except segment.

Reply:
Yeah, indeed. It seems it's not needed at all any more.